CITATION.cff

```diff
@@ -26,6 +26,8 @@ authors:
   given-names: Rashida
 - family-names: Katiyar
   given-names: Ankita
+- family-names: Ko
+  given-names: Elijah
```

Comment:
Curious is this is following the TSC now?

Reply:
Yes I think we should not add new authors until they're part of the TSC

Reply:
Meaning that we should remove Jo and Elijah from this PR